test/test_stateless.py

```diff
@@ -44,43 +45,47 @@ def _run_call_with_mock_module(self, module, device='cpu', prefix=''):
         # the parameters represent an identity function contrary to the
         # existing params in module. So here we expect the result to be the
         # same as the input if the weight swapping went well.
-        res = stateless.functional_call(module, parameters, x)
+        res = functional_call(module, parameters, x)
         self.assertEqual(x, res)
         # check that the weight remain unmodified
         cur_weight = to_check.l1.weight
         cur_buffer = to_check.buffer
         self.assertEqual(cur_weight, prev_weight)
         self.assertEqual(cur_buffer, prev_buffer)
 
-    def test_functional_call(self):
+    @parametrize("functional_call", [torch.func.functional_call, stateless.functional_call])
+    def test_functional_call(self, functional_call):
```

Comment:
nit: What exactly does the test name for this look like? If it has any special characters in it (like `test_functional_call_<function functional_call of torch.func>` then we want to parametrize with subtest to give it a sane name.

e.g.
```
 @parametrize_test('weight_layout', [
     subtest(torch.strided, name='weightStrided'),
     subtest(torch.sparse_coo, name='weightCOO'),
     subtest(torch.sparse_csr, name='weightCSR'),
     subtest(torch.sparse_csc, name='weightCSC'),
     # TODO: addmm: computation on CPU is not implemented for Strided + Strided @ SparseBsr
     # subtest(torch.sparse_bsr, name='weightBSR'),
     # subtest(torch.sparse_bsc, name='weightBSC'),
 ])
 ```

Reply:
TIL about subtest! Thanks!